node: assert/strict, fs/promises, and perf_hooks modules cannot be loaded by require()

**Describe the bug**

`assert/strict`, `fs/promises`, and `perf_hooks` modules have already been implemented, but cannot be loaded by `require()`.

**To Reproduce** Steps to reproduce the behavior:

1. Create a file named `sample.ts` as follows:

```ts:sample.ts
import { createRequire } from "https://deno.land/std@0.103.0/node/module.ts";

const require = createRequire(import.meta.url);
const fsPromises = require("fs/promises");
console.log(fsPromises);
```

2. Run the `sample.ts`:

```ts
$ deno run -A --unstable sample.ts
error: Uncaught Error: Cannot find module 'fs/promises'
Require stack:
- /home/uki00a/ghq/github.com/uki00a/deno_std/sample.ts
      const err = new Error(message) as Error & {
                  ^
    at Function._resolveFilename (https://deno.land/std@0.103.0/node/module.ts:273:19)
    at Function._load (https://deno.land/std@0.103.0/node/module.ts:380:29)
    at Module.require (https://deno.land/std@0.103.0/node/module.ts:133:21)
    at require (https://deno.land/std@0.103.0/node/module.ts:1172:16)
    at file:///home/uki00a/ghq/github.com/uki00a/deno_std/sample.ts:4:20
```

**Expected behavior**

No error should occur.

**Desktop (please complete the following information):**

- OS: Ubuntu 20.04
- Version

```
deno 1.12.2 (release, x86_64-unknown-linux-gnu)
v8 9.2.230.14
typescript 4.3.5
```

--- a/node/module.ts
+++ b/node/module.ts
@@ -22,6 +22,7 @@
 import "./global.ts";
 
 import nodeAssert from "./assert.ts";
+import nodeAssertStrict from "./assert/strict.ts";
 import nodeBuffer from "./buffer.ts";
 import nodeCrypto from "./crypto.ts";
 import nodeConsole from "./console.ts";
@@ -29,8 +30,10 @@ import nodeConstants from "./constants.ts";
 import nodeChildProcess from "./child_process.ts";
 import nodeEvents from "./events.ts";
 import nodeFS from "./fs.ts";
+import nodeFSPromises from "./fs/promises.ts";
 import nodeOs from "./os.ts";
 import nodePath from "./path.ts";
+import nodePerfHooks from "./perf_hooks.ts";
 import nodeQueryString from "./querystring.ts";
 import nodeStream from "./stream.ts";
 import nodeStringDecoder from "./string_decoder.ts";
@@ -618,6 +621,10 @@ function createNativeModule(id: string, exports: any): Module {
 }
 
 nativeModulePolyfill.set("assert", createNativeModule("assert", nodeAssert));
+nativeModulePolyfill.set(
+  "assert/strict",
+  createNativeModule("assert/strict", nodeAssertStrict),
+);
 nativeModulePolyfill.set("buffer", createNativeModule("buffer", nodeBuffer));
 nativeModulePolyfill.set(
   "constants",
@@ -633,9 +640,17 @@ nativeModulePolyfill.set(
   createNativeModule("events", nodeEvents),
 );
 nativeModulePolyfill.set("fs", createNativeModule("fs", nodeFS));
+nativeModulePolyfill.set(
+  "fs/promises",
+  createNativeModule("fs/promises", nodeFSPromises),
+);
 nativeModulePolyfill.set("module", createNativeModule("module", Module));
 nativeModulePolyfill.set("os", createNativeModule("os", nodeOs));
 nativeModulePolyfill.set("path", createNativeModule("path", nodePath));
+nativeModulePolyfill.set(
+  "perf_hooks",
+  createNativeModule("perf_hooks", nodePerfHooks),
+);
 nativeModulePolyfill.set(
   "querystring",
   createNativeModule("querystring", nodeQueryString),

--- a/node/module_test.ts
+++ b/node/module_test.ts
@@ -86,15 +86,19 @@ Deno.test("requireModuleWithConditionalExports", () => {
 Deno.test("requireNodeJsNativeModules", () => {
   // Checks these exist and don't throw.
   require("assert");
+  require("assert/strict");
   require("buffer");
   require("child_process");
+  require("console");
   require("constants");
   require("crypto");
   require("events");
   require("fs");
+  require("fs/promises");
   require("module");
   require("os");
   require("path");
+  require("perf_hooks");
   require("querystring");
   require("stream");
   require("string_decoder");
@@ -105,14 +109,12 @@ Deno.test("requireNodeJsNativeModules", () => {
 
   // TODO(kt3k): add these modules when implemented
   // require("cluster");
-  // require("console");
   // require("dgram");
   // require("dns");
   // require("http");
   // require("http2");
   // require("https");
   // require("net");
-  // require("perf_hooks");
   // require("readline");
   // require("repl");
   // require("sys");

